test/integration/testUtil.js

```diff
@@ -72,6 +72,28 @@ module.exports.executeCmdAsync = function (connection, sqlText, binds = undefine
   });
 };
 
+module.exports.executeCmdAsync = executeCmdAsync;
+
+/**
+ * Drop tables one by one if exist - any connection error is ignored
+ * @param connection Connection
+ * @param tableNames string[]
+ * @return {Promise<void>}
+ */
+module.exports.dropTablesIgnoringErrorsAsync = async (connection, tableNames) =>
+{
+  for (let tableIdx in tableNames)
+  {
+    const tableName = tableNames[tableIdx];
+    try
+    {
+      await executeCmdAsync(connection, `DROP TABLE IF EXISTS ${tableName}`);
+    }catch (e){
```

Comment:
I guess, you should reformat code in this line

Reply:
Yes, thanks - I have formatted it according to eslint config